### DISCUSSION TITLE
We need to provide BoltzTraP interface

### DISCUSSION THREAD

**Discussion by hongriTianqi** [deepmodeling/abacus-develop#1599] [General]
### Background

BoltzTraP/BoltzTraP2 are open source codes for calculating charge/phonon transport properties of materials. Adding abacus interface of BoltzTraP/BoltzTraP2 will increase functionality of abacus.

### Describe the solution you'd like

Adding the abacus interface of BoltzTraP/BoltzTraP2.

**Comment by mohanchen** [comment]
What are the details/procedures for adding the BoltzTrap interface? How long will it take?
